STL_Extension/include/CGAL/Handle.h

```diff
@@ -51,6 +51,12 @@ class Handle
       PTR->count++;
     }
 
+    Handle(Handle&& x) noexcept
+      : PTR(x.PTR)
+    {
+      x.PTR = static_cast<Rep*>(0);
+    }
+
```

Comment:
@GilesBathgate it is this block

Reply:
Ah, I added this code as suggested by @mglisse 's comment:

>This violates the rule of 0/3/5, which is more common than your static analysis tool. If we add the move assignment, we should also add
>    Handle(Handle&& x) noexcept : PTR(x.PTR) { x.PTR = 0; }

I must admit, I don't know what the correct implementation should be for a move constructor here.

Reply:
That constructor is fine. If the git-bisection of Maxime is right, that new constructor has triggered a bug, but is only indirectly responsible for that bug.